www/front_src/src/components/pollerMenu/index.js

```diff
@@ -75,6 +77,10 @@ class PollerMenu extends Component {
 
   render() {
     const { data } = this.props;
+    const { entries } = this.props.navigationData;
+
+    // check if poller configuration page is allowed
+    const allowPollerConfiguration = entries.includes('60901')
```

Comment:
this is the corresponding id of the poller configuration page
it is the legacy routing system of centreon... waiting more readable routes

Reply:
Ok, so the comment should be enought here
But i guess this magic number is used at many places
There is a lot of value to encapsulate those ID at a single place, in self explained constants
Especially since people are going to refactor that code as you said